doc/md/atlas-schema/projects.mdx

```diff
@@ -122,6 +122,30 @@ atlas schema apply --env local
 Will run the `schema apply` command against the database that is defined for the `local`
 environment.
 
+:::info Unnamed Environments
+It is possible to define an `env` block whose name is dynamically set during command execution using the `--env` flag.
```

Comment:
```suggestion
It is possible to define an `env` block which's name is dynamically set during command execution using the `--env` flag.
```

Reply:
Do you mean `which its`? cc @rotemtam, my Grammarly.